File: MovieCatalogQw/app/src/main/java/com/unlistedi/moviecatalogqw/ui/movies/MoviesViewModel.java
package com.unlistedi.moviecatalogqw.ui.movies;

import androidx.lifecycle.ViewModel;

import com.unlistedi.moviecatalogqw.data.MovieEntity;
import com.unlistedi.moviecatalogqw.utils.DataGenerator;

import java.util.ArrayList;

public class MoviesViewModel extends ViewModel {
    private ArrayList<MovieEntity> movies = DataGenerator.getMovies();

    public ArrayList<MovieEntity> getMovies() {
        return movies;
    }
}

File: MovieCatalogueKu/app/src/main/java/com/unlistedi/moviecatalogueku/ui/movies/MoviesViewModel.kt
package com.unlistedi.moviecatalogueku.ui.movies

import android.app.Activity
import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.unlistedi.moviecatalogueku.data.MovieEntity
import com.unlistedi.moviecatalogueku.utils.DataGenerator

class MoviesViewModel : ViewModel() {
    val movies : ArrayList<MovieEntity> = DataGenerator.getMovies()
}